### DISCUSSION TITLE
Dual spec price

### DISCUSSION THREAD

**Discussion by Kirishiki** [chromiecraft/chromiecraft#3587] [Suggestions]
Could the dual spec price be lowered by half to encourage more people get it for more flexible raiding/dungeoning experience so we dont have too many dpses who queue as tanks or heals then failing and saying dual spec was too expensive; ending up wasting peoples time.

**Comment by Amrx2** [comment]
Hi Kirishiki, dual spec remains as one of our primary gold sinks and is already reduced from it's usual price. The price increases with each bracket due to the ease of farming gold at higher levels. Due to the above we won't be altering the cost. 

Cheers, Construct.
